### PR TITLE
Admissions process WIP

### PR DESCRIPTION
**Pull request by nebrius**
This is pretty early stage, still a WIP. I'm posting so we can start gathering feedback.

### CODE REVIEW

**Comment by Trott** [comment]
Do other WGs have membership policies we can look at for comparison?

**Comment by nebrius** [comment]
@Trott Here's my understanding of how each current WG adds new members:

- Website: open open source admissions, e.g. make a content PR
- Streams: no info
- Build: no info
- Tracing: no info
- Hardware: hop in gitter and ask (documented this way), we have no accept/reject mechanism, we accept all
- i18n, each language is different. I can go through each one if you want, but there's a lot.
- Evangelism: open open source admissions
- Roadmap: no info
- Docker: open open source admissions
- Addon API: no info


@mikeal, does that seem accurate?

**Comment by sup** [comment]
afaik, the process for i18n is _mostly_ asking nicely. but i suppose it differs for the more active and populated ones

**Comment by ashleygwilliams** [comment]
hey i'm a little late to the party, but this is a great start @nebrius. looking forward to getting it in place.

**Comment by Trott** [comment]
In terms of group target size, the TSC targets 6-12 people. That doesn't mean we have to do that too, but worth noting/thinking about.

From https://github.com/nodejs/TSC/blob/master/TSC-Charter.md#section-4-establishment-of-the-tsc:

> TSC memberships are not time-limited. There is no fixed size of the TSC. However, the expected target is between 6 and 12, to ensure adequate coverage of important areas of expertise, balanced with the ability to make decisions efficiently.

**Comment by juliepagano** [comment]
This was briefly discussed in [today's meeting](https://github.com/nodejs/inclusivity/blob/master/meetings/2015-12-18.md), so it is worth reviewing some of the notes in there around mention of #32.

**Comment by nebrius** [comment]
Just a note from the changes, a draft of the form is live at https://nebrius.typeform.com/to/dsvEs5, so please consider that as part of this PR too.

**Comment by Charlotteis** [comment]
> Just a note from the changes, a draft of the form is live at https://nebrius.typeform.com/to/dsvEs5, so please consider that as part of this PR too.

@nebrius Filled out the form and I think there were a good range of questions. You can also change the CoC link to the merged one now rather than the PR :)

**Comment by Charlotteis** [comment]
This all reads pretty well to me.

One question: if I were to be rejected after application, is there scope to apply again in the future? If so, it'd be cool to mention that in this doc.

**Comment by nebrius** [comment]
> Filled out the form and I think there were a good range of questions.

I got the response, thanks! I have a question for you, was it obvious some questions were optional, especially the marginalized group reporting questions? Typeform has a small indicator, but I was contemplating adding `(optional)` to the end of the fields to make it more explicit. What do you think?

> You can also change the CoC link to the merged one now rather than the PR :)

Good catch, thanks! It's updated now.

> One question: if I were to be rejected after application, is there scope to apply again in the future? If so, it'd be cool to mention that in this doc.

That's actually a really good question. I'm tempted to say yes, with a buffer period, so that rejected candidates have to wait, say 3 months or something. What does everyone else think?

**Comment by Charlotteis** [comment]
> was it obvious some questions were optional, especially the marginalized group reporting questions

Nope, I assumed all of them were necessary (I didn't have a problem with that, but can see why You'd want to make it optional). Maybe you could indicate that in the question title.

**Comment by nebrius** [comment]
I updated the type form to make the questions more obvious if they're optional, thanks!

**Comment by nebrius** [comment]
@Charlotteis also, just to make sure I'm not misreading anything, is your form submission an official request to join the WG, was it for testing/debugging purposes, etc? The admissions process is still a work in progress of course, but if your request is official, we can try out the process For Realz (tm) and see how it works :)

**Comment by Charlotteis** [comment]
@nebrius yeah! If you accept me I'd like to be the 'collaborator' type of folk mentioned by Julie in another Issue (defining roles), but I'd love to be a part of this :)

**Comment by Charlotteis** [comment]
> I updated the type form to make the questions more obvious if they're optional, thanks!

much clearer now :)

**Comment by ashleygwilliams** [comment]
thoughts on getting this merged? or should we wait for #65

**Comment by nebrius** [comment]
Should we move the file into docs, say `docs/POLICY_ADMISSIONS.md`? Then squash. I think it'll be ready to go then, unless anyone else has objections?

**Comment by ashleygwilliams** [comment]
seems like a plan to me @nebrius

**Comment by nebrius** [comment]
@ashleygwilliams, ok should be good to go!

File: docs/POLICY_ADMISSIONS.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+NOTE: this is an early stage WIP, and it's contents will most likely be split up into multiple locations before landing
```
Comment (ashleygwilliams) [inline docs/POLICY_ADMISSIONS.md]: can we get rid of this line? i know it is still early stage but this is what we are working with for now so let's own it.

File: docs/POLICY_ADMISSIONS.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+If you are interested in becoming a member of the Inclusivity Working Group, please submit your application at https://nebrius.typeform.com/to/dsvEs5.
```
Comment (ashleygwilliams) [inline docs/POLICY_ADMISSIONS.md]: perhaps hide this URL? e.g. `[please submit your application here](URLgoeshere)`

**Comment by ashleygwilliams** [comment]
eeee sorry 2 small nits

**Comment by nebrius** [comment]
Look better? I can squash if everything else looks good

**Comment by ashleygwilliams** [comment]
wfm squash away!

**Comment by nebrius** [comment]
squash, nom nom

**Comment by ashleygwilliams** [comment]
one day i'll convince you to use active present and not past tense :stuck_out_tongue_closed_eyes:

**Comment by nebrius** [comment]
lol, 😛

**Pull request merged**